backend/collect/api.py

```diff
@@ -41,3 +43,22 @@ def get_object(self):
         self.check_object_permissions(self.request, collection)
         return collection
 
+
+class CollectionRecordsView(RDFView):
+    '''
+    View the records inside a collection
+    '''
+
+    def get_graph(self, request: Request, collection: str, **kwargs) -> Graph:
+        collection_uri = URIRef(collection)
+
+        if not collection_exists(collection_uri):
+            raise NotFound('Collection does not exist')
+
+        collection_obj = EDPOPCollection(collection_graph(collection_uri), collection_uri)
+
+        g = Graph()
+        g += collection_obj._class_triples()
+        g += EDPOPCollection.records._stored_triples(collection_obj)
```

Comment:
The convention for methods starting with an underscore is that they are not supposed to be called, or even known, outside their own class. Should these methods be private in the first place, and if so, isn't there some kind of public interface that you can use instead?

Reply:
Yeah, I think I initially envisioned these as "private" methods but they probably should just be public. You don't really need the method here, though.

Reply:
I you don't really need it (which of the two, by the way?), then why is it invoked?